docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -724,6 +725,37 @@ directly or used in conditional statements.
 - `$file_size`
   : Size of the file for this row.
 
+### Replace tables
+
+The connector supports replacing a table, in case the table already exists, as
+an atomic operation. Atomic table replacement creates a new snapshot with the
+new table definition and keeps the existing table history. See
+{ref}`delta-lake-history-table` in metadata tables.
+
+To replace a table, use `CREATE OR REPLACE TABLE` and/or `CREATE OR REPLACE
+TABLE AS`. For more information on creating tables, see {doc}`CREATE TABLE
+</sql/create-table>` and {doc}`CREATE TABLE AS </sql/create-table-as>`. These
+statements do not support changing table formats defined by the `type` table
+property.
+
+After replacement, the table definition is completely new and separate from the
+old table.
+
+For example, a table `example_table` can be replaced by a completely new
+definition:
+
+```
+CREATE TABLE example_table (
+    a BIGINT,
+    b DATE,
+    c BIGINT)
+WITH (partitioning = ARRAY['a']);
```

Comment:
confirm this is valid table property in delta lake

Reply:
It should be `partitioned_by`. 
https://github.com/trinodb/trino/blob/23e5ec5b3d30182548d90636a8fbaec9c82ffcfe/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java#L41